Keas.Mvc/Helpers/DbInitializer.cs

```diff
@@ -39,7 +40,19 @@ public void Initialize()
             CreateSystemPermission(new SystemPermission { Role = admin, User = scott });
             CreateSystemPermission(new SystemPermission { Role = admin, User = cal });
             CreateSystemPermission(new SystemPermission { Role = admin, User = jason });
-            CreateSystemPermission(new SystemPermission { Role = admin, User = nabil });
+            CreateSystemPermission(new SystemPermission { Role = admin, User = nabil });
+
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "Service Tag"                 , Description = "Service Tag"                             , TeamId = null});
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "Hostname"                    , Description = "Hostname"                                , TeamId = null });
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "IP Address"                  , Description = "IP Address"                              , TeamId = null });
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "Mac Address"                 , Description = "Mac Address"                             , TeamId = null });
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "OS"                          , Description = "Operating System"                        , TeamId = null });
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "Owner"                       , Description = "The person responsible for the equipment", TeamId = null }); //Maybe we will add this as a field?
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "Phone Number"                , Description = "Phone Number"                            , TeamId = null });
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "PrePurchasing Request Number", Description = "PrePurchasing Request Number"            , TeamId = null });
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "Warranty Expiration Date"    , Description = "Warranty Expiration Date"                , TeamId = null });
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "Purchase Date"               , Description = "Date was purchased"                      , TeamId = null });
+            CreateGlobalAttributeKey(new EquipmentAttributeKey { Key = "External Link"               , Description = "URL to more info"                        , TeamId = null });
```

Comment:
quick thoughts on this list.  Overall pretty great.  Ideas:
1 - Might as well turn OS into Operating System since the other ones are not acronyms.
2 - PrePurchasing Request Number -> Order Number, since many people buy through aggiebuy and other systems.  Not sure about this one exactly (leave it alone? maybe have >1 request number area? Change to order link?).
3- MAC should be capitalized just like IP since it's an acronym.